torch/utils/mkldnn.py

```diff
@@ -202,7 +202,7 @@ def forward(self, x):
         return y
 
 def to_mkldnn(module, dtype=torch.float):
-    assert dtype in [torch.float, torch.bfloat16], \
+    assert dtype in [torch.float, torch.bfloat16, torch.half], \
         "MKLDNN only support float or bfloat16 path now"
```

Comment:
should also add torch.float16 to the error message? :)

Reply:
Thanks for your comments. I will fix it in https://github.com/pytorch/pytorch/pull/99498.